### ISSUE TITLE
Links must have discernible text

### ISSUE THREAD

**Issue by christinach** [pulibrary/lux-design-system#223]
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Ensures links have discernible text

### Severity
High

### WCAG Success Criterion
[SC 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

[SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

### User impact
Users do not understand the purpose of each link so they can decide whether they want to follow the link.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:

    Element does not have text that is visible to screen readers
    aria-label attribute does not exist or is empty
    aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
    Element has no title attribute

And fix the following:

Element is in tab order and does not have accessible text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)

Update links in footer to have discernible text
